app/Cargo.toml

```diff
@@ -22,11 +24,14 @@ attestation_agent = { path = "../", default-features = false }
 async-trait = "0.1.56"
 
 [build-dependencies]
-tonic-build = "0.7.2"
+tonic-build = { version = "0.7.2", optional = true }
+ttrpc-codegen = { version = "0.4.1", optional = true }
 shadow-rs = "0.11.0"
```

Comment:
Can you help to issue another PR to remove shadow-rs? It has been removed from ocicrypt-rs and image-rs.

Reply:
Removed!